src/fileinfo/file_presentation/json_presentation.cpp

```diff
@@ -514,12 +514,19 @@ void WriteSignature(JsonPresentation::Writer& writer, const DigitalSignature& si
 void JsonPresentation::presentCertificates(Writer& writer) const
 {
 
-	if(!fileinfo.certificateTable)
+	if (!fileinfo.certificateTable)
 	{
 		return;
 	}
-
-	writer.String("digitalSignatures");
+	bool isOutside = fileinfo.certificateTable->isOutsideImage;
+	if (isOutside)
+	{
+		writer.String("digitalSignatures");
+	}
+	else
+	{
+		writer.String("invalidSignatures");
```

Comment:
I don't feel like this really needs to be here. I propose getting rid of these invalid signatures.

Reply:
Okay, updated so the signatures inside the image are ignored and not presented in the output.